examples/jira/jira_review_groups.py

```diff
@@ -66,7 +66,7 @@ def get_inactive_users(groups):
     return inactive_users_list
 
 
-def exclude_inactive_users(groups):
+def exclude_inactive_users(groups: list[dict]):
```

Comment:
It makes sense to add smth using at least TypedDict or even [pydantic model](https://docs.pydantic.dev/latest/concepts/models/#basic-model-usage) as better option
```
from pydantic import BaseModel

class User(BaseModel):
    name: str
    ...

class Group(BaseModel):
    group_name: str
    users: list[User]
    ...

def exclude_inactive_users(groups: list[Group]):
```